packages/components/src/VirtualizedList/HeaderResizable/HeaderResizable.component.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Draggable from 'react-draggable';
+import { ConsumerVirtualizedList } from '../virtualizedListContext';
+import headerResizableCssModule from './HeaderResizable.scss';
+import { getTheme } from '../../theme';
+import getDefaultT from '../../translate';
+
+const theme = getTheme(headerResizableCssModule);
+
+const HeaderResizableContent = ({ customRender, ...rest }) => {
+	if (customRender) {
+		return customRender;
+	} else if (rest.label) {
+		return rest.label;
+	}
+	return null;
+};
+
+export class HeaderResizable extends React.Component {
+	state = {
+		resizing: false,
+	};
+
+	setResizing = () => {
```

Comment:
You should take the value to set instead of toggle. This is not really safe to rely on user start/stop event + toggle.

Reply:
Ok let's be safe here